load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -760,17 +761,16 @@ func (plan *actorLoadTestIncrementPlan) shouldIncrementSupplierCount(
 }
 
 // addActor populates the actors's amount to stake and accAddress using the
-// keyName provided in the corresponding provisioned actors slice.
-func (s *relaysSuite) addActor(keyName string, actorStakeAmount sdk.Coin) *accountInfo {
-	keyRecord, err := s.txContext.GetKeyring().Key(keyName)
-	require.NoError(s, err)
-
-	accAddress, err := keyRecord.GetAddress()
+// address provided in the corresponding provisioned actors slice.
+func (s *relaysSuite) addActor(actorAddress string, actorStakeAmount sdk.Coin) *accountInfo {
+	accAddress, err := sdk.AccAddressFromBech32(actorAddress)
+	keyRecord, err := s.txContext.GetKeyring().KeyByAddress(accAddress)
 	require.NoError(s, err)
+	require.NotNil(s, keyRecord)
 
 	return &accountInfo{
 		accAddress:    accAddress,
```

Comment:
Do we still need both `accAddress` and `address` or is there a way to just keep one?